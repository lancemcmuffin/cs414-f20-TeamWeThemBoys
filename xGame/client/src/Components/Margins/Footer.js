import React, {Component} from "react";

export default Footer = () => {
    return (
        <p>add footer data</p>
    );
}