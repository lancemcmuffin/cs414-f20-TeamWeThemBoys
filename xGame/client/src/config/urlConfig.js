/**
 * @module urlConfig.js
 * @description exports URL constants used by services
 */

const localhostURL = "http://localhost:8080";
const serverURL = "tbd";

const serviceURL = localhostURL;

export const matchServiceURL = `${serviceURL}/match`;
export const userServiceURL = `${serviceURL}/user`;
export const messageServiceURL = `${serviceURL}/message`;
export const gameServiceURL = `${serviceURL}/game`;
