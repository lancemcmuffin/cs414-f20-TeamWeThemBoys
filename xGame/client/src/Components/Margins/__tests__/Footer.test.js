import "../../../enzyme.config.js";
import React from "react";
import { shallow } from "enzyme";

import Footer from "../Footer";

function testInitialState() {
  const app = shallow(<Footer />);
  expect(app.html()).toMatchSnapshot();
}

test("Testing footer initial state", testInitialState);
