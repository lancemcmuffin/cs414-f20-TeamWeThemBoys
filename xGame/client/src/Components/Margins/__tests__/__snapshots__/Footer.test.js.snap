// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing footer initial state 1`] = `"<div class=\\"footer\\"><p>This site built by Team We Them Boys</p></div>"`;
